services/github/src/pr-allowed.js

```diff
@@ -1,81 +0,0 @@
-const yaml = require('js-yaml');
-
-const DEFAULT_POLICY = 'collaborators';
-
-async function prAllowed(options) {
```

Comment:
Explanation: This function return a boolean, but there are three cases: `true`, `false` and `neither 😳` (if true, we should check if the author of the PR is collaborator, if false we should comment* to that effect and exit, if neither - we should just exit)

* in case it's PR.opened - in other cases we can just exit